[2023-04-20 09:46:03,066] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:46:03] "GET /previsoes HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:46:04,385] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:46:04] "GET /previsoes HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:46:19,103] WARNING get_produto() L104  Erro ao buscar previsao 'Maresias', Previsao não encontrada na base :/ - call_trace=C:\Users\leticia.gerola\Desktop\Pessoal\pos_grad\mvp_sprint_1\back_end\app.py L104 
[2023-04-20 09:46:19,109] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:46:19] "[33mGET /previsao?nome_praia=Maresias HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:46:30,290] WARNING del_previsao() L135  Erro ao deletar previsao #'Teste', Previsao não encontrada na base :/ - call_trace=C:\Users\leticia.gerola\Desktop\Pessoal\pos_grad\mvp_sprint_1\back_end\app.py L135 
[2023-04-20 09:46:30,292] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:46:30] "[33mDELETE /previsao?nome_praia=Teste HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:46:31,644] WARNING del_previsao() L135  Erro ao deletar previsao #'Teste', Previsao não encontrada na base :/ - call_trace=C:\Users\leticia.gerola\Desktop\Pessoal\pos_grad\mvp_sprint_1\back_end\app.py L135 
[2023-04-20 09:46:31,645] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:46:31] "[33mDELETE /previsao?nome_praia=Teste HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:46:43,490] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:46:43] "[33mGET /produtos HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:47:07,342] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:47:07] "[33mPOST /produto HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:47:17,381] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:47:17] "[33mPOST /produto HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:47:22,610] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:47:22] "[33mOPTIONS /produto?nome=camburi HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:31,190] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5002
 * Running on http://192.168.15.25:5002 (Press CTRL+C to quit) - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:33,651] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:33] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:33,660] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:33] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:33,743] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:33] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:33,747] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:33] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:33,749] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:33] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:33,945] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:33] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:52,739] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:52] "POST /previsao HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:52:53,645] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:52:53] "POST /previsao HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:01,251] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:01] "GET /previsoes HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:02,252] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:02] "GET /previsoes HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:15,967] WARNING get_produto() L104  Erro ao buscar previsao 'Teste', Previsao não encontrada na base :/ - call_trace=C:\Users\leticia.gerola\Desktop\Pessoal\pos_grad\mvp_sprint_1\back_end\app.py L104 
[2023-04-20 09:53:15,969] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:15] "[33mGET /previsao?nome_praia=Teste HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:25,374] WARNING del_previsao() L135  Erro ao deletar previsao #'Teste', Previsao não encontrada na base :/ - call_trace=C:\Users\leticia.gerola\Desktop\Pessoal\pos_grad\mvp_sprint_1\back_end\app.py L135 
[2023-04-20 09:53:25,375] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:25] "[33mDELETE /previsao?nome_praia=Teste HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:26,716] WARNING del_previsao() L135  Erro ao deletar previsao #'Teste', Previsao não encontrada na base :/ - call_trace=C:\Users\leticia.gerola\Desktop\Pessoal\pos_grad\mvp_sprint_1\back_end\app.py L135 
[2023-04-20 09:53:26,721] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:26] "[33mDELETE /previsao?nome_praia=Teste HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:37,419] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:37] "[33mGET /produtos HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:53:59,969] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:53:59] "[33mPOST /produto HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:54:11,436] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:54:11] "[33mPOST /produto HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 09:54:16,306] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 09:54:16] "[33mOPTIONS /produto?nome=camburi HTTP/1.1[0m" 404 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 10:03:33,413] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5002
 * Running on http://192.168.15.25:5002 (Press CTRL+C to quit) - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 10:03:34,405] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 10:03:34] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 10:03:34,435] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 10:03:34] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 10:03:34,558] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 10:03:34] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 10:03:34,578] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 10:03:34] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-04-20 10:03:34,588] INFO _log() L224  127.0.0.1 - - [20/Apr/2023 10:03:34] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
